bedrock/firefox/views.py

```diff
@@ -334,6 +335,7 @@ def firefox_partners(request):
         request, 'firefox/partners/index.html', 'firefox.partners.index', template_vars, form_kwargs)
 
 
+# Note: this function is also used to determine display for dev-whatsnew
```

Comment:
Would it be worthwhile renaming this function to something more generic, to avoid confusion in the whatsnew view itself?